Sources/Action/Action+Extensions.swift

```diff
@@ -21,3 +21,14 @@ public extension Action where Input == Void {
         return execute(())
     }
 }
+
+extension CompletableAction {
+    /// Emits everytime work factory completes.
+    public var completions: Observable<Void> {
```

Comment:
Can't this be a Completable ? 
```suggestion
    public var completions: Completable {
```

Reply:
Not really! `Completable` will emit a single complete event while we want a next event with each _completed_ execution. Something similar to `errors`

Reply:
@freak4pc all good here `completions: Observable<Void>` is fine. My only suggestion is to change the `extension CompletableAction` to `public extension CompletableAction` and make the property `var completions: Observable<Void>`. this is just align the new `Swift` convention. All looks good to go. Thanks